src/coprocessor/codec/mysql/json/json_cast.rs

```diff
@@ -46,7 +43,7 @@ impl Json {
             Json::I64(d) => d as f64,
             Json::U64(d) => d as f64,
             Json::Double(d) => d,
-            Json::String(ref s) => convert::bytes_to_f64(ctx, s.as_bytes())?,
+            Json::String(ref s) => convert_bytes_to_f64(ctx, s.as_bytes())?,
```

Comment:
Maybe we need to refine this function later. It is wired again for a data type function relies on a convert function.

Reply:
The `json_cast.rs` will be removed and all related logic will be moved to `Convert` trait (will use a trait to organize convert logic after the PR merge, there are no more convert functions).